ledger/participant-integration-api/src/main/scala/platform/apiserver/ApiServices.scala

```diff
@@ -95,6 +97,8 @@ private[daml] object ApiServices {
       seedService: SeedService,
       managementServiceTimeout: Duration,
       enableSelfServiceErrorCodes: Boolean,
+      checkOverloaded: TelemetryContext => Option[state.SubmissionResult] =
+        _ => None, // Used for Canton rate-limiting
```

Comment:
I would prefer not to default the value at two different levels. Instead I would provide the default in SandboxServer.